specifyweb/frontend/js_src/lib/components/Toolbar/TreeRepair.tsx

```diff
@@ -75,10 +75,12 @@ export function TreeSelectDialog({
                 : hasTreeAccess(treeName, 'read')
             )
             .map((treeName) => {
-              const treeDefinition = deserializeResource(
-                getTreeDefinitions(treeName)[0].definition
-              );
-              return [treeName, treeDefinition] as const;
+              const treeDefinitions = getTreeDefinitions(treeName);
+              const defaultTreeDefinition =
+                treeDefinitions.length > 0
+                  ? deserializeResource(treeDefinitions[0].definition)
+                  : undefined;
+              return [treeName, defaultTreeDefinition] as const;
```

Comment:
(optional) print console.warn if there are no tree definitions for a given tree?
such console log messages will appear in the crash reports so could be useful of some bugs occur because tree defs are missing - though if we are already including tree defs in the crash reports, then this would be redundant.